<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#authenticate_label {
	font-family:Yu Gothic UI ;
	font-size: 12pt;
	font-weight: 600;
}

#frame QPushButton {
	padding: 2px;
	font-weight: 550;
	color: white;
	border: 5px solid #e1dfdf;
}

#back_btn {
	border-radius: 17px;
	font-size: 11pt;
	background-color: #efa913;
}

#next_btn {
	border-radius: 17px;
	font-size: 11pt;
	background-color: #27a37e;
}

#end_btn {
	border-radius: 10px;
	font-size: 11pt;
	background-color: #F24942;
}

#bypass_btn {
	font-size: 8pt;
	background-color: #4070f4;
}

#back_btn:disabled, #next_btn:disabled, #bypass_btn:disabled {
	background-color: #e1dfdf;
	color: gray;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="authenticate_label">
        <property name="text">
         <string>Authentication Phase</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bypass_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string> Bypass Authentication (-100 coins)</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../resources/icons/key.svg</normaloff>../resources/icons/key.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>  Back</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../resources/icons/arrow-left.svg</normaloff>../resources/icons/arrow-left.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Next  </string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../resources/icons/arrow-right.svg</normaloff>../resources/icons/arrow-right.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="end_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string> End Simulation</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../resources/icons/x-octagon.svg</normaloff>../resources/icons/x-octagon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
